app/scripts/directives/bindService.js

```diff
@@ -71,6 +71,7 @@ function BindService($scope,
                                         .concat(replicaSets)
                                         .concat(statefulSets);
       ctrl.applications = _.sortByAll(apiObjects, ['metadata.name', 'kind']);
```

Comment:
OK, I can update https://github.com/openshift/origin-web-common/blob/master/src/components/binding/bindServiceForm.js to use these values.